packages/import-bundle/src/index.js

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
imo, this is the concern of agoric-sdk since this package doesn't have any `tsc`.

I'll approve per your inter-repo maintenance judgement but I think there should be a comment like,
```
XXX omit from typecheck by consumers
TODO https://github.com/endojs/endo/issues/1254
```